GameData/000_FilterExtensions Configs/SubCategories_LBP.cfg

```diff
@@ -0,0 +1,56 @@
+SUBCATEGORY
+{
+	name = Modern Ships
+	icon = boat
+	FILTER
+	{
+		CHECK
+		{
+			type = title
+			value = modern,dg,medium,tri,tumble,vette
```

Comment:
`type = title`
title is a check for that value at any position. These short/common id's would be very error prone outside the intended category with the current filtering. Would prefer something that was less reliant on the category to work

eg.
```
type = tag
value = modernShip
```
with all relevant parts having the appropriate tag